app/controllers/api/v1/studies_controller.rb

```diff
@@ -85,10 +85,132 @@ def show
       end
 
       # return JSON of the study, all study file objects, and any options values needed for the upload wizard
+      # also includes feature flags, with study flags taking precedence over user flags
+      swagger_path '/studies/{id}/file_info' do
+        operation :get do
+          key :tags, [
+            'Studies'
+          ]
+          key :summary, 'Get Study info'
+          key :description, 'Gets attribute, file, feature flag, and configuration information about a single Study'
+          key :operationId, 'study_file_info_path'
+          parameter do
+            key :name, :id
+            key :in, :path
+            key :description, 'ID of Study to fetch'
```

Comment:
`/studies/{id}/...` implies that this value is a study ID. Not sure if description is required, but if not, is this too much noise?

Reply:
Technically, it will support either a Mongo BSON ID, or an accession.  See [here](https://github.com/broadinstitute/single_cell_portal_core/blob/development/app/controllers/api/v1/concerns/study_aware.rb#L8) for how studies are loaded in the API.